packages/se-productivity-metrics/se-productivity-metrics.js

```diff
@@ -0,0 +1,178 @@
+Metrics = function(timeout) {
+
+  colors.enabled = true; // To color Terminal outputs
+
+  /**
+  Initializing options
+  **/
+  if (typeof timeout === "number") {
+    this.options = {
+      timeout: timeout,
+      headers: { 'user-agent': 'meteor.js' }
+    }
+  } else {
+    this.options = {
+      // timeout: 5000,
+      headers: { 'user-agent': 'meteor.js' },
+      params: {
+        client_id: 'd275d9504a71d088dd47',
+        client_secret: 'c35b2ad3a90df7837926251664026d8e8e438915'
```

Comment:
@omarkamal95 you sure this should be published ? or should be passed in settings or env variable